opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -44,7 +44,7 @@ message InstrumentationLibraryMetrics {
   repeated Metric metrics = 2;
 }
 
-// Defines a Metric which has one or more timeseries.
+// Metric represents timeseries produced by a telemetry system.
```

Comment:
```suggestion
// Metric represents a timeseries produced by a telemetry system.
```
Did you mean a singular or plural form of timeseries?

Reply:
Singular. Thanks.